test/integration/features/basic.feature

```diff
@@ -172,6 +172,8 @@ Feature: Basic
      """
      exposed
      """
+    When status code of HTTP request to service "ruby-ex" of namespace "ruby" is equal to "200"
```

Comment:
I guess what is missing regarding my initial comment is to parameterise the path. Atm, you only hit "/" of the app. What if you wanted to hit "/welcome" or you want to verify that there is a welcome page "/welcome"?

    And status code of HTTP request "/" of service "ruby-ex" in namespace "ruby" is equal to "200"

You then just need to extract this additional path parameter and pass it to _verifyHTTPResponse_. You already pass something, namely the empty string (aka the root "/"). You have all the pieces in place. The step definition just needs to capture one more variable.